Mention Portals + Testing Library workaround on README 💬 (#134)
nickcolley/jest-axe: README.md

> Note: If you're using `react testing library` <9.0.0 you should be using the
> [`cleanup`](https://testing-library.com/docs/react-testing-library/api#cleanup) method. This method removes the rendered application from the DOM and ensures a clean HTML Document for further testing.

> In case you're using this library to verify some component that relies on [React Portals](https://reactjs.org/docs/portals.html#gatsby-focus-wrapper), make sure you supply [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:

Reviewer: ```suggestion
If you're using [React Portals](https://reactjs.org/docs/portals.html), use the [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:

```

```js
it('should work with React Portals as well', async () => {

>   const { baseElement } = render(<App/>)

Reviewer: I'm wondering if we could do this for all examples?

Is there any downside to always using this baseElement?

Reviewer: If this did work, we dont need to mention portals at all, just update the existing examples.

Author: I see these days on the [`testing-library` docs](https://testing-library.com/docs/react-testing-library/example-intro) they use `screen` on their examples most of the time instead of `container`. 

Maybe we could use that instead of `baseElement` on the examples, I need to double check though it doesn't break the tests with portals though 👍🏽

  const results = await axe(baseElement)
  
  expect(results).toHaveNoViolations()
})
```

> Otherwise, `jest-axe` won't be able to perform its clean-up properly and throw an error.

Reviewer: ```suggestion
```

Reviewer: I think we can leave this out and focus on just mentioning portals.
